cfme/infrastructure/pxe.py

```diff
@@ -35,12 +35,12 @@
     fields=[
         ('name_text', Input('name')),
         ('log_protocol', Select("//select[@id='log_protocol']")),
-        ('uri_text', Input('uri')),
```

Comment:
Yes , just changes order hence the URI field gets sufficient time to load else it shows error element not found